entrypoint.sh

```diff
@@ -27,9 +27,9 @@ fi
 echo "Loading private key for JWT from: $JWT_PRIVATE_KEY_FILE_PATH"
 echo "Loading public key for JWT from: $JWT_PUBLIC_KEY_FILE_PATH"
 
-if [ -z $SALT ]; then
+if [ -n "$SALT" ]; then
 	SALT_PARAMETER=",\"salt\":\"$SALT\""
-elif [ -f $SALT_FILE ]; then
+elif [ -n "$SALT_FILE" ]; then
```

Comment:
Are you sure we should change it like that? -f checks if the file actually exists and this was the intention here.